assets/src/stories-editor/blocks/amp-story-page/index.js

```diff
@@ -49,7 +49,7 @@ const schema = {
 		default: '[]',
 	},
 	overlayOpacity: {
-		default: 50,
+		default: 100,
```

Comment:
Note that this will likely cause invalidation issues in some templates, it needs updating in the template files, too.

Reply:
Actually, it looks like for the Travel templates no validation errors appear. It would for Fandom templates though, in case these should get merged before this PR.

Reply:
Updated templates in ffda3a97589671e737aca8f289b1fe8797dc2360

Reply:
Thanks!